Paket.sln

```diff
@@ -34,7 +34,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "content", "content", "{8E6D
 		docs\content\index.md = docs\content\index.md
 		docs\content\lock-file.md = docs\content\lock-file.md
 		docs\content\nuget-dependencies.md = docs\content\nuget-dependencies.md
-		docs\content\paket-config-file.md = docs\content\paket-config-file.md
```

Comment:
yeah, I treat config as a command so it's now generated from the HelpTexts class.
Eventually we'll be able to fallback to separate md file for each command when this PR is merged